### ISSUE TITLE
Ubuntu 16.04

### ISSUE THREAD

**Issue by nazar-pc** [numixproject/numix-plymouth-theme#1]
There are some fundamental issues with Ubuntu 16.04, I guess it happened couple of months ago on Plymouth update when directories were changed.
Anyone here capable of understanding what happens and how to fix it? Xenial is going to be released soon.

**Comment by morph027** [comment]
Directory for themes now is _/usr/share/plymouth/themes_. Tried it, but still does not work ;) I'll try to debug later using plymouth-x11 and [plymouth-preview](http://www.khattam.info/plymouth-preview-a-tool-to-preview-your-plymouth-theme-2010-11-19.html)

**Comment by morph027** [comment]
Ok, the lame way is just to create a symlink:

```
sudo ln -s /lib/plymouth/themes/numix /usr/share/plymouth/themes/numix
```

Cool way is to adjust the path during building the deb file and also replace things inside _numix.plymouth_

```
[Plymouth Theme]
Name=Numix
Description=Numix Plymouth theme.
ModuleName=script

[script]
ImageDir=/usr/share/plymouth/themes/numix
ScriptFile=/usr/share/plymouth/themes/numix/numix.script
```

**Comment by morph027** [comment]
My fix for now:

[numix-plymouth-theme_20150828-f6dfe13_amd64.deb.zip](https://github.com/numixproject/numix-plymouth-theme/files/508396/numix-plymouth-theme_20150828-f6dfe13_amd64.deb.zip)

``` bash
mkdir /tmp/foo
docker run --rm -it -v /tmp/foo:/build -w /build ubuntu:xenial /bin/bash
apt-get update
apt-get -qq install ruby ruby-dev build-essential git -y
gem install fpm
git clone https://github.com/numixproject/numix-plymouth-theme
cd numix-plymouth-theme

cat > lib/plymouth/themes/numix/numix.plymouth << EOF 
[Plymouth Theme]
Name=Numix
Description=Numix Plymouth theme.
ModuleName=script

[script]
ImageDir=/usr/share/plymouth/themes/numix
ScriptFile=/usr/share/plymouth/themes/numix/numix.script
EOF

cat > after-install.sh << EOF
update-alternatives --install /usr/share/plymouth/themes/default.plymouth default.plymouth /usr/share/plymouth/themes/numix/numix.plymouth 300
update-initramfs -u -k all
EOF

cat > before-remove.sh << EOF
update-alternatives --remove default.plymouth /usr/share/plymouth/themes/numix/numix.plymouth
update-initramfs -u -k all
EOF

fpm -s dir -t deb \
--prefix "/usr/share/plymouth/themes" \
--name numix-plymouth-theme \
--version "20150828-f6dfe13" \
--maintainer "Numix Theme" \
--description "Numix Plymouth Theme" \
--url "https://github.com/numixproject/numix-plymouth-theme" \
--after-install after-install.sh \
--before-remove before-remove.sh \
--depends plymouth-label \
-C lib/plymouth/themes/ .
```

**Comment by khurshid-alam** [comment]
@PaoloRotolo Can you fix this as per above comment?

**Comment by sqc1999** [comment]
+1 for this
But it seems to be hard to fix the problem and keep backward compatibility at the same time.

**Comment by mrmeszaros** [comment]
@paolorotolo any progress on this?

**Comment by Foggalong** [comment]
This is fixed in [v1.1](https://github.com/numixproject/numix-plymouth-theme/releases/tag/v1.1) - legacy support is achieved through commented line in makefile.
